pygmt/tests/test_plot.py

```diff
@@ -446,3 +447,28 @@ def test_plot_datetime():
     y = [8.5, 9.5]
     fig.plot(x, y, style="i0.2c", pen="1p")
     return fig
+
+
+@pytest.mark.mpl_image_compare(filename="test_plot_sizes.png")
+def test_plot_deprecate_sizes_to_size(data, region):
+    """
+    Make sure that the old parameter "sizes" is supported and it reports an
+    warning.
+
+    Modified from the test_plot_sizes() test.
+    """
+    fig = Figure()
+    with warnings.catch_warnings(record=True) as w:
+        fig.plot(
+            x=data[:, 0],
+            y=data[:, 1],
+            sizes=0.5 * data[:, 2],
+            region=region,
+            projection="X10c",
+            style="cc",
+            color="blue",
+            frame="af",
+        )
+        assert len(w) == 1
+        assert issubclass(w[0].category, FutureWarning)
```

Comment:
Actually, would it be better to use `pytest.warns` here? See https://docs.pytest.org/en/6.2.x/warnings.html#warns.

Reply:
Might need to remove the `import warnings` line at the top.

```suggestion
    with pytest.warns(expected_warning=FutureWarning) as record:
        fig.plot(
            x=data[:, 0],
            y=data[:, 1],
            sizes=0.5 * data[:, 2],
            region=region,
            projection="X10c",
            style="cc",
            color="blue",
            frame="af",
        )
        assert len(record) == 1  # check that only one warning was raised
```